htdocs/commande/class/api_commande.class.php

```diff
@@ -106,12 +107,12 @@ function get($id='',$ref='', $ref_ext='', $ref_int='')
      * @url     GET     /order/list
      * @return  array   Array of order objects
      */
-    function getList($mode=0, $sortfield = "s.rowid", $sortorder = 'ASC', $limit = 0, $page = 0) {
+    function getList($mode=0, $societe = "", $sortfield = "s.rowid", $sortorder = 'ASC', $limit = 0, $page = 0) {
```

Comment:
If you introduce a new param of a function, you must add it at end. Otherwise this break compatibility.